packages/hardhat-core/src/internal/hardhat-network/provider/node.ts

```diff
@@ -1692,3 +1764,15 @@ Hardhat Network's forking functionality only works with blocks from at least spu
     });
   }
 }
+
+function getCommon(networkId: number, blockNumber: number): Common | undefined {
+  try {
+    const common = Common.forCustomChain(networkId, {});
```

Comment:
I think we don't want a custom one here, but rather a normal/standard `Common`. You can get one with `new Common({chain: networkId})`.

Reply:
I still don't understand the difference between them :sweat_smile:  But, anyway, fixed on a19fea87b32607120e01ba82959aad20c8120e51